server/data/hmppsRestClient.ts

```diff
@@ -0,0 +1,260 @@
+import { Readable } from 'stream'
```

Comment:
we should already have something for this?

Reply:
**suggestion:** use [rest client](https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/server/data/restClient.ts) instead and remove multer dependency.

We've got to be a bit careful of copying across old idioms from CVL

Reply:
Ya this one: server/data/restClient.ts